### ISSUE TITLE
Missing library annotations for java.util.concurrent.atomic package

### ISSUE THREAD

**Issue by kanha95** [typetools/checker-framework#1155]
Nullness checker currently raises false warnings against 5 classes of java.util.concurrent.atomic package. Namely AtomicMarkableReference<V> , AtomicReference<V> , AtomicReferenceArray<E>, AtomicReferenceFieldUpdater<T,V>, AtomicStampedReference<V>.

For Example:

```java
import org.checkerframework.checker.nullness.qual.*;
import java.util.concurrent.atomic.*;

class Test{
  
    public static void main(String... args){
      
        AtomicReference ar=new AtomicReference(null);
        System.out.println(ar);
        
    }
}
```
Output using nullness checker: 
```
found : null
required: ? extends @Initialized @NonNull Object
at line 8
```
Output at runtime: null

**Comment by kanha95** [comment]
@mernst  hello professor. I would like to know why this issue got an invalid tag ? Is this issue insignificant?

**Comment by mernst** [comment]
Hmm, I had a lengthy comment that seems to have disappeared.  I'm sorry my action was confusing!

Your test case contains raw types, which are bad style and are explicitly discouraged by the Checker Framework manual.  Just running regular javac issues a warning, and people shouldn't use pluggable type-checking unless they are willing to use regular Java type-checking.

When i changed the test case to use proper parameterized types, such as `AtomicReference<@Nullable String>`, then no warning was produced.  So, there does not seem to be a bug here.
